float32 status
float32 tujuan_x
float32 tujuan_y
float32 enc_left
float32 enc_right
float32 th